### PR TITLE
19 compute set intersection

### PR DESCRIPTION
**Pull request by stellaljung**
Please review the test formatting. I included the testing in the main.cpp file due to this [Crow testing tutorial](https://crowcpp.org/master/guides/testing/). I only tested response codes in the backend because there was not enough detail in the Crow framework of how to get the body of the request. If there is a better way of catch/throwing vs cout/cerr please add in the pull request.

### CODE REVIEW

**Review by csirianni — CHANGES_REQUESTED** [review]
Commit messages look amazing!!! Thanks so much.

File: frontend/app/handlers.ts [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
-            method: "GET",
+        const response = await fetch("http://localhost:18080/passwords", {
+            method: "POST",
             mode: "cors",
             headers: {
-                "Access-Control-Allow-Headers": "*",
+                "Access-Control-Allow-Headers": "*", // cors setting
+                "Content-Type": "application/json"
             },
+            body: password
```
[the remark this answers is not in the document]
Comment (stellaljung) [reply frontend/app/handlers.ts:11]: Changing body to be a JSON object will be a later issue when we have multiple things sent in the json body request, but for now body is standard for the fetch API and can not be changed.

File: backend/test/password.cpp [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #define CATCH_CONFIG_MAIN
 #include <regex>
 #include <catch2/catch_test_macros.hpp>
-#include "../src/password.hpp"
+#include "password.hpp"
```
Comment (ni-jessica) [inline backend/test/password.cpp:4]: **question**: I'm wondering why this doesn't need relative import.

**Review by ni-jessica — APPROVED** [review]

File: backend/test/password.cpp (hunk 2 above)
Comment (ni-jessica) [reply backend/test/password.cpp:4]: nevermind i just didn't read the group chat, i get it now!

**Review by csirianni — APPROVED** [review]

**Pull request merged**